modules/ocf_decal/manifests/init.pp

```diff
@@ -14,10 +14,24 @@
     system  => true,
   }
 
-  file { '/opt/ocfdecal':
-    ensure  => directory,
-    owner   => ocfdecal,
-    group   => ocfdecal,
-    require => User['ocfdecal'];
+  file {
+    '/opt/ocfdecal':
+      ensure  => directory,
+      owner   => ocfdecal,
+      group   => ocfdecal,
+      require => User['ocfdecal'];
+    '/etc/decal_mysql.conf':
+      source  => 'puppet:///private/mysql.conf',
+      owner   => ocfdecal,
+      group   => ocfstaff,
+      mode    => '0660',
```

Comment:
I'd probably make this `0440` and maybe put a notice at the top of the file or something, since you don't want people to be editing it on the server instead of on the puppet private share (in the vein of other files we do this with).